tests/e2e/plugins/index.js

```diff
@@ -0,0 +1,11 @@
+const fs = require('fs-extra');
+const path = require('path');
+
+function getConfigurationByFile(file) {
+  const pathToConfigFile = path.resolve('..', 'EXPLORER', 'tests', 'e2e', 'config', `${file}.json`);
```

Comment:
whith this you actually override your directory with repo, and expect it to be `EXPLORER`